run-build-functions.sh

```diff
@@ -675,6 +699,8 @@ cache_artifacts() {
   cache_home_directory ".boot" "boot dependencies"
   cache_home_directory ".composer" "composer dependencies"
   cache_home_directory ".wasmer/cache", "wasmer cache"
+  cache_home_directory ".cargo/registry" "rust deps"
+  check_rust_target_cacheability && cache_home_directory "repo/target" "rust compile output"
```

Comment:
where is `repo/target` coming from?
i think in order to cache something from the user's repository you need to use `cache_cwd_directory`

Reply:
I don't know I just got this code from a previous PR. `target` is definitely the name of the compilation directory (a bit like `node_modules`).

So should I replace by this?

```
check_rust_target_cacheability && cache_home_directory "$(cache_cwd_directory)"/target "rust compile output"
```

Reply:
OH I see!! Nevermind what I said.

```
check_rust_target_cacheability && cache_cwd_directory target "rust compile output"
```

And I will move it up there with the others

Reply:
yup! great

Reply:
Fixed by db4ab67